src/main/java/io/fusionauth/domain/UsageDataConfiguration.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024-2024, FusionAuth, All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package io.fusionauth.domain;
+
+import java.util.Objects;
+
+import com.inversoft.json.JacksonConstructor;
+
+/**
+ * Config for Usage Data / Stats
+ *
+ * @author Lyle Schemmerling
+ */
+public class UsageDataConfiguration extends Enableable {
+  public int numberOfDaysToRetain = 366;
```

Comment:
This is an interesting default. Why not `365`?

Reply:
There are 366 days in a leap year.